deploy/runtime/defaults/Flat Plate PV_Residential.txt

```diff
@@ -391,6 +391,18 @@ batt_max_power
 1
 1
 0
+batt_salvage_percentage
+1
+1
+1
+1
+0
+batt_salvage_value
+1
```

Comment:
Why are the other defaults changing? I get the ones you added but did the module and inverter change?